src/containers/treePage/ducks/types.ts

```diff
@@ -0,0 +1,101 @@
+export interface SiteProps {
+  siteId: number;
+  blockId: number;
+  lat: number;
+  lng: number;
+  city: string;
+  zip: string;
+  address: string;
+  entries: SiteEntry[];
+}
+
+export interface SiteEntry {
+  id: number;
+  username: string;
+  updatedAt: number;
+  treePresent?: boolean;
+  status?: string;
+  genus?: string;
+  species?: string;
+  commonName?: string;
+  confidence?: string;
+  diameter?: number;
+  circumference?: number;
+  coverage?: string;
+  pruning?: string;
+  condition?: string;
+  discoloring?: boolean;
+  leaning?: boolean;
+  constrictingGrate?: boolean;
+  wounds?: boolean;
+  pooling?: boolean;
+  stakesWith?: boolean;
+  stakesWithout?: boolean;
+  light?: boolean;
+  bicycle?: boolean;
+  bagWith?: boolean;
+  bagWithout?: boolean;
+  tape?: boolean;
+  suckerGrowth?: boolean;
+  siteType?: string;
+  sidewalkWidth?: string;
+  siteWidth?: string;
+  siteLength?: string;
+  material?: string;
+  raisedBed?: string;
+  fence?: string;
+  trash?: string;
+  wires?: string;
+  grate?: string;
+  stump?: string;
+  treeNotes?: string;
+  siteNotes?: string;
+}
+
+export const SiteEntryNames: Record<string, string> = {
+  id: 'Id',
+  username: 'Username',
+  updatedAt: 'Updated At',
+  treePresent: 'Tree Present',
+  status: 'Status',
+  genus: 'Genus',
+  species: 'Species',
+  commonName: 'Common Name',
+  confidence: 'Confidence',
+  diameter: 'Diameter',
+  circumference: 'Circumference',
+  coverage: 'Coverage',
+  pruning: 'Pruning',
+  condition: 'Condition',
+  discoloring: 'Discoloring',
+  leaning: 'Leaning',
+  constrictingGrate: 'Constricting Grate',
+  wounds: 'Wounds',
+  pooling: 'Pooling',
+  stakesWith: 'Stakes With',
+  stakesWithout: 'Stakes Without',
+  light: 'Light',
+  bicycle: 'Bicycle',
+  bagWith: 'Bag With',
+  bagWithout: 'Bag Without',
+  tape: 'Tape',
+  suckerGrowth: 'Sucker Growth',
+  siteType: 'Site Type',
+  sidewalkWidth: 'Sidewalk Width',
+  siteWidth: 'Site Width',
+  siteLength: 'Site Length',
+  material: 'Material',
+  raisedBed: 'Raised Bed',
+  fence: 'Fence',
+  trash: 'Trash',
+  wires: 'Wires',
+  grate: 'Grate',
+  stump: 'Stump',
+  treeNotes: 'Tree Notes',
+  siteNotes: 'Site Notes',
+} 
+
+export interface TreeCare {
```

Comment:
📌 Will get the following information for stewardship activities (minus duration): https://docs.c4cneu.com/sftt/sftt-api-spec/#get-stewardship-activities-by-site - will have to make a selector or string format function to get a neat string date and message